src/service/core/config/helpers.py

```
@@ -198,7 +198,7 @@ def patch_configs(
             except ValueError as _:
                 raise osmo_errors.OSMOUserError(
                     f'Bucket mode {bucket_config.mode} is not valid. Valid modes are '
-                    f'{", ".join([member.value for member in connectors.BucketMode])}')
+                    f'{', '.join([member.value for member in connectors.BucketMode])}')
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)